deploy/operator.yaml

```diff
@@ -68,6 +71,80 @@ spec:
           path: /run/dbus/system_bus_socket
           type: Socket
 ---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nmstate-handler-worker
```

Comment:
Make sense to also change the other daemonset to nmstate-handler-master or it's problematic with upgrades and similar stuff ?

Reply:
It is problematic because of the upgrade. We will need to take care of deleting the old one.

Reply:
We have a mechanism for that in CNAO though :) and downtime of knmstate caused by the upgrade is not an issue. Let's keep the naming nice.

Reply:
We can rename it in a follow up though.